src/CBLUserAgent.hh

```diff
@@ -93,12 +93,24 @@ static string getDistroInfo() {
 using stringstream = std::stringstream;
 using alloc_slice = fleece::alloc_slice;
 
+static std::string getCCommit(){
+    static const std::regex r(R"([\w-]+$)");
```

Comment:
The size of a git commit hash is not likely to change, so why not just use the last X chars of this?  It's probably pretty noisy to include the entire hash inside.  Do you think it could be shortened?

Reply:
The first part of CBLITE_SOURCE_ID is a formatted timestamp which, I guess, will maintain format - not likely to change. Thus, the below should do 
`std::string ss(CBLITE_SOURCE_ID);
  std::string cCommit = ss.substr(20);`

@borrrden please let me know if I should change to the above